src/components/stepforms/widgets/ColumnSelector.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
I think this whole file should be removed now that we have the `sync` property on `ColumnPicker`.